canonicalwebteam/blog/wordpress_api.py

```diff
@@ -65,7 +62,11 @@ def get_article(slug, tags=[], tags_exclude=[]):
 
     response = api_session.get(url)
 
-    return process_response(response)
+    data = process_response(response)
+    if len(data) > 0:
```

Comment:
Actually, this does rather beg the question, why have `get_article` at all? If it's just going to use the same `posts` endpoint as `get_articles`, it's probably a more faithful representation of the Wordpress API to just provide the `get_articles` function, and let the client just grab the first article if that's all it wants.

Thoughts?

Reply:
I think that this is due to the return from that endpoint. 
Making the slug not optional is key to keeping this function I feel. 
When consuming this **API** it seems nicer to have 2 separate functions, instead of overloading one with parameters.